zubhub_frontend/zubhub/src/views/search_results/SearchResults.jsx

```diff
@@ -37,7 +37,13 @@ import Project from '../../components/project/Project';
 import styles from '../../assets/js/styles/views/search_results/searchResultsStyles';
 import commonStyles from '../../assets/js/styles';
 
-const useStyles = makeStyles(styles);
+const useStyles = makeStyles({
+  ...styles,
```

Comment:
* `styles` is actually a function, it shouldn't work. `const styles = theme => ({`
* Instead of duplicating styles, just apply two classnames for the html element.

Reply:
Not understanding

Reply:
this file exports a function https://github.com/yokwejuste/zubhub/blob/fix/spacing/zubhub_frontend/zubhub/src/assets/js/styles/views/search_results/searchResultsStyles.js

we can only spread object or an array, the usage of styles is wrong.

Reply:
Great, thanks, will do some research on this

Reply:
you could to something like this.

```javascript
const useCommonStyles = makeStyles(theme => ({
  someClassName: {}
}}))
const useSpecificStyles = makeStyles(theme => ({
someClassName: {},
two: {}
}}))

const MyComponent = () => {
   const commonClassNames = useCommonStyles();
   const specificClassNames = useSpecificStyles();
   return <div className={cslx(commonClassNames.someClassName, specificClassNames.someClassName, specificClassNames.two)}>text</div>
}